lib/config_for/capistrano.rb

```diff
@@ -70,14 +70,18 @@ def remote_file(task)
           file = shared_path.join(t.name).to_s.shellescape
 
           on roles(target_roles) do
-            unless test "[ -f #{file} ]"
+            if override? || !test("[ -f #{file} ]")
               info "Uploading #{prerequisite_file} to #{file}"
               upload! File.open(prerequisite_file), file
             end
           end
 
         end
       end
+
+      def override?
+        ENV['OVERRIDE_CONFIG_FILES'] == 'true'
```

Comment:
I think we can just use `options` passed to this Task no ? 
This is meant to be a library, so I'd say it shouldn't use ENV variables.